nibabel/filebasedimages.py

```diff
@@ -0,0 +1,569 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+''' Simple interfaces for images'''
+
+try:
+    basestring
+except NameError:  # python 3
+    basestring = str
```

Comment:
from .externals.six import string_types

And replace `basestring` with `string_types` throughout.